### PR TITLE
Display OIDs consistently

### LINKED ISSUE #1504: Inconsistent display formats for OIDs
The spec currently displays X.509 OIDs in at least three different ways, and after PR #1491 there will be four:

- [§8.2. Packed Attestation Statement Format](https://w3c.github.io/webauthn/#sctn-packed-attestation):

  ```
  - If |attestnCert| contains an extension with OID 1.3.6.1.4.1.45724.1.1.4 (`id-fido-gen-ce-aaguid`) verify [...]
  ```

- [§8.3. TPM Attestation Statement Format](https://w3c.github.io/webauthn/#sctn-tpm-attestation):

  ```
  - If |aikCert| contains an extension with OID `1 3 6 1 4 1 45724 1 1 4` (id-fido-gen-ce-aaguid) verify [...]
  ```

- [§8.4.1. Android Key Attestation Statement Certificate Requirements](https://w3c.github.io/webauthn/#sctn-key-attstn-cert-requirements):

  ```
  Android Key Attestation [=attestation certificate=]'s <dfn>android key attestation certificate extension
  data</dfn> is identified by the OID "1.3.6.1.4.1.11129.2.1.17", and its schema [...]
  ```

- (PR #1491) [§8.8. Apple Anonymous Attestation Statement Format](https://w3c.github.io/webauthn/#sctn-apple-anonymous-attestation):

  ```
  4. Let Apple anonymous attestation CA generate an X.509 certificate for the
    [=credential public key=] and include the |nonce| as a certificate extension
    with OID ( 1.2.840.113635.100.8.2 ). [...]
  ```

We should standardize on one display format. I propose we use "code quotes" like this for both the OID and, if available, its name:

```
- If |attestnCert| contains an extension with OID `1.3.6.1.4.1.45724.1.1.4` (`id-fido-gen-ce-aaguid`) verify [...]
```

### PR DESCRIPTION
**Pull request by emlun**
Fixes #1504.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1508.html" title="Last updated on Nov 4, 2020, 8:12 PM UTC (2ad9e5f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1508/87ca0fe...2ad9e5f.html" title="Last updated on Nov 4, 2020, 8:12 PM UTC (2ad9e5f)">Diff</a>

### CODE REVIEW

**Comment by emlun** [comment]
Right, there's also this in [§8.3.1. TPM Attestation Statement Certificate Requirements](https://w3c.github.io/webauthn/#sctn-tpm-cert-requirements) which I'm not sure what it means:

>The Extended Key Usage extension MUST contain the "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" OID.

Should we do anything about that too?

**Comment by bdewater** [comment]
Sounds like we should, it is [`2.23.133.8.3`](http://oid-info.com/get/2.23.133.8.3).

**Comment by equalsJeffH** [comment]
> it is 2.23.133.8.3

I'd do something like:

> The Extended Key Usage extension MUST contain the "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" ( `2.23.133.8.3` ) OID.

**Review by equalsJeffH — APPROVED** [review]
LGTM

**Review by akshayku — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -4894,7 +4894,7 @@ TPM [=attestation certificate=] MUST have the following fields/extensions:
 - The Subject Alternative Name extension MUST be set as defined in [[!TPMv2-EK-Profile]] section 3.2.9.
 
 - The Extended Key Usage extension MUST contain the
-    "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" OID.
+    "joint-iso-itu-t(2) internationalorganizations(23) 133 tcg-kp(8) tcg-kp-AIKCertificate(3)" (`2.23.133.8.3`) OID.
```
Comment (bdewater) [inline index.bs:4897]: Nit: in all the examples above the numerical numerical dot notation comes first, the human-readable notation (if available) second. It seems best to use that here as well. The numerical notation is probably what you'll want as an implementer and one can always figure out (parts of) the human-readable notation with some internet searching.

Comment (emlun) [reply index.bs:4897]: Fixed in #1513, thanks! :smile:

**Pull request merged**